### PR TITLE
Handle new possible iOS in-app-purchases

### PR DESCRIPTION
**Pull request by phillipthelen**
This adds handling for the new iOS IAP tiers. also changes the buyGems function to handle different amounts of gems.

### CODE REVIEW

**Comment by crookedneighbor** [comment]
@vIiRuS Will this be compatible with the current ios app? And the old mobile app?

**Comment by crookedneighbor** [comment]
Actually, the old mobile app didn't do gem purchases, did it?

**Comment by negue** [comment]
Only the old android app has gem purchases

**Comment by phillipthelen** [comment]
As it also has the old identifier for 20 gems (but awards 21) it is also compatible with the old/current ios version

File: website/src/controllers/payments/index.js [hunk 1]
```diff
@@ -137,7 +137,8 @@ exports.cancelSubscription = function(data, cb) {
 }
 
 exports.buyGems = function(data, cb) {
-  var amt = data.gift ? data.gift.gems.amount/4 : 5;
+  var amt = data.amount ? data.amount : 5;
```
Comment (crookedneighbor) [inline website/src/controllers/payments/index.js]: This should probably be: `var amt = data.amount || 5;`

**Comment by crookedneighbor** [comment]
One style comment, otherwise, :+1:

**Comment by phillipthelen** [comment]
Wasn't aware that you could do that. Changed it :)

**Pull request merged**